app/controllers/EmailController.java

```diff
@@ -19,13 +20,17 @@
 @Controller("emailController")
 public class EmailController extends BaseController {
 
+    private BridgeConfig bridgeConfig;
```

Comment:
Yes, I agree.  It's really ugly when injection and inheritance get mixed up.  It's actually the intention of this refactoring to move away from inheritance.  Maybe I should be more aggressive and split out part of the BaseController that uses other services into a new service?  As far as I can tell, it's mainly about managing session.  Maybe call it SessionService.